evm/contracts/AccessGrantsV1.sol

```diff
@@ -1,11 +1,14 @@
 // SPDX-License-Identifier: MIT
-pragma solidity =0.8.19;
+pragma solidity ^0.8.20;
```

Comment:
bump and pin

Reply:
Skipped this for now, since this is only the implementation part and I've readied everything with this code.

Happy to follow-up. It'll be a good exercise of the upgrade mechanism!